hobbit-gui/gui-serverbackend/src/main/java/de/usu/research/hobbit/gui/rest/ExperimentsResources.java

```diff
@@ -65,28 +67,81 @@
         if (Application.isUsingDevDb()) {
             return getDevDb().queryExperiments(ids, challengeTaskId);
         } else {
+            // get user
+            UserInfoBean userInfo = InternalResources.getUserInfoBean(sc);
+            List<SystemBean> userSystems = PlatformControllerClientSingleton.getInstance().requestSystemsOfUser(userInfo.getPreferredUsername());
```

Comment:
You could define the variable in that line and run the request later on if the information is needed.

```
List<SystemBean> userSystems = null;
```